src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -144,20 +147,22 @@ function formReadReceiptsTooltip(
   const recipientString = readBy.length === 1 ? displayRecipientType : `${displayRecipientType}s`;
   return (
     <FlexBox className={cssClass("readReceipts--container")} alignItems="center" justify="end">
-      <Tooltip
-        className={cssClass("readReceipts--tooltip")}
-        content={readReceiptString}
-        placement={"top"}
-        textAlign={"left"}
-      >
-        <FlexBox>
-          <Checkmark className={cssClass("readReceipts--icon")} />
-          <span className={cssClass("readReceipts--text--desktop")}>
-            Read by {readReceiptCount} {recipientString}
-          </span>
-          <span className={cssClass("readReceipts--text--mobile")}>{readReceiptCount}</span>
-        </FlexBox>
-      </Tooltip>
+      <div onMouseOver={onReadReceiptsHover}>
```

Comment:
I had thought it would make more sense to put the `<div` directly around the the stuff inside the Tooltip instead of around the Tooltip as well. that way the idea if they hover over the trigger (the child of the tooltip) then we would record a log

Reply:
Makes sense, I'll change this rn

Reply:
Just wondering, how would this interact if the hover function was wrapped around the tooltip? How would this behave differently relative to the tooltip?

Reply:
To be honest im not certain... i would have to test it out to check. I worried that it would double count if someone hovered over the trigger and then again over the tooltip itself